packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -637,126 +502,83 @@ async function handleCommitMessagePreference(commitMessageFlag) {
  *
  * It performs the following actions:
  *  - TODO - Add a list of what this function does
  */
 async function createRedwoodApp() {
-  // Introductory message
-  tui.drawText(
-    [
-      `${RedwoodStyling.redwood('-'.repeat(66))}`,
-      `${' '.repeat(16)}🌲⚡️ ${RedwoodStyling.header(
-        'Welcome to RedwoodJS!'
-      )} ⚡️🌲`,
-      `${RedwoodStyling.redwood('-'.repeat(66))}`,
-    ].join('\n')
-  )
-
   const cli = yargs(hideBin(process.argv))
     .scriptName(name)
-    .usage('Usage: $0 <project directory> [option]')
-    .example('$0 newapp')
-    .option('typescript', {
-      alias: 'ts',
+    .usage('Usage: $0 <project directory>')
+    .example('$0 my-redwood-app')
+    .version(version)
+    .option('yes', {
+      alias: 'y',
       default: null,
       type: 'boolean',
-      describe: 'Generate a TypeScript project.',
+      describe: 'Skip prompts and use defaults',
     })
     .option('overwrite', {
       default: false,
       type: 'boolean',
       describe: "Create even if target directory isn't empty",
     })
-    .option('telemetry', {
-      default: true,
+    .option('typescript', {
+      alias: 'ts',
+      default: null,
       type: 'boolean',
-      describe:
-        'Enables sending telemetry events for this create command and all Redwood CLI commands https://telemetry.redwoodjs.com',
+      describe: 'Generate a TypeScript project',
     })
     .option('git-init', {
       alias: 'git',
       default: null,
       type: 'boolean',
-      describe: 'Initialize a git repository.',
+      describe: 'Initialize a git repository',
     })
     .option('commit-message', {
       alias: 'm',
       default: null,
       type: 'string',
-      describe: 'Commit message for the initial commit.',
-    })
-    .option('yes', {
-      alias: 'y',
-      default: null,
-      type: 'boolean',
-      describe: 'Skip prompts and use defaults.',
+      describe: 'Commit message for the initial commit',
     })
-    .version(version)
-
-  const _isYarnBerryOrNewer = isYarnBerryOrNewer()
-
-  // Only permit the yarn install flag on yarn 1.
-  if (!_isYarnBerryOrNewer) {
-    cli.option('yarn-install', {
-      default: null,
+    .option('telemetry', {
+      default: true,
       type: 'boolean',
-      describe: 'Install node modules. Skip via --no-yarn-install.',
+      describe:
+        'Enables sending telemetry events for this create command and all Redwood CLI commands https://telemetry.redwoodjs.com',
     })
-  }
 
   const parsedFlags = cli.parse()
 
+  tui.drawText(
+    [
+      `${RedwoodStyling.redwood('-'.repeat(66))}`,
+      `${' '.repeat(16)}🌲⚡️ ${RedwoodStyling.header(
+        'Welcome to RedwoodJS!'
+      )} ⚡️🌲`,
+      `${RedwoodStyling.redwood('-'.repeat(66))}`,
+    ].join('\n')
+  )
+
   // Extract the args as provided by the user in the command line
   // TODO: Make all flags have the 'flag' suffix
   const args = parsedFlags._
-  const yarnInstallFlag =
```

Comment:
This is the only change I'm not thrilled by here.

I know you've already tried to make installation work even with yarn > 1, but do you think you've learned anything new since then that might make it possible now? It *sounds* easy enough to just cd into the directory and run `yarn install` in there. But I can't remember what issues you ran into when you tried last time

Reply:
Nothing new to report. The issue there is only yarn > 1 as you said, and it's something about esbuild failing to do a post install step. Only happens on linikers other than the `node-modules` one. Yarn just doesn't seem to pick up the settings in the project when using any variant of `execa` for whatever reason; it can't like change its settings and binary mid-process.

I think this PR can go without this change. I probably just included it because I was trying to change things I wasn't a fan of at the same time but if those require more discussion don't want to bundle them together here.

Reply:
Added back in https://github.com/redwoodjs/redwood/pull/9786/commits/ce1537154008c22fcb4439847623663e726257b8